.github/workflows/ci.yml

```diff
@@ -0,0 +1,265 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - dev
+      - stable
+    paths-ignore:
+      - 'documentation/**'
+      - '**.md'
+  pull_request:
+    branches:
+      - dev
+      - stable
+    paths-ignore:
+      - 'documentation/**'
+      - '**.md'
+
+jobs:
+  build-and-test:
+    name: Build and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust: [nightly, beta, stable]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install LLVM and Clang (Windows)
+      uses: KyleMayes/install-llvm-action@32c4866ebb71e0949e8833eb49beeebed48532bd
+      if: matrix.os == 'windows-latest'
+      with:
+        version: "11.0"
+        directory: ${{ runner.temp }}/llvm
+
+    - name: Set LIBCLANG_PATH (Windows)
+      run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
+      if: matrix.os == 'windows-latest'
+
+    - name: Install ${{ matrix.rust }}
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ matrix.rust }}
+        override: true
+
+    - name: Cache cargo state
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-cargo
+      with:
+        path: |
+          ~/.cargo/registry
+          ~/.cargo/git
+        key: ${{ matrix.os }}-${{ matrix.rust }}-build-${{ env.cache-name }}-${{ hashFiles('**/Cargo.toml') }}
```

Comment:
I think that `build` part is a relic from the previous version ? It should be covered by the cache name now.

Reply:
We need it for more granularity in the caching. But it was in the wrong position here, so thanks for pointing that out!